### PR TITLE
[sitecore-jss-cli] RAV fetchWith uses old GraphQl endpoint path in package.json

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Description / Motivation
Set `/sitecore/api/graph/edge` graphqlEndpointPath as default one when use `jss create`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

### CODE REVIEW

**Review by ambrauer — CHANGES_REQUESTED** [review]
We also need to handle the `<app/>` `graphQLEndpoint` in [the Sitecore config](https://github.com/Sitecore/jss/blob/3d2ad76fd4505fc6f3e9429414c033bff6dcedc1/packages/sitecore-jss-cli/src/create/index.ts#L97). 

I wonder if it's time to retire the old "default/conventional" values and require all samples use the `replaceName` parameter ("name replacement mode") like nextjs does. We could remove quite a bit of code in `sitecore-jss-cli/create` in the process.

File: packages/sitecore-jss-cli/src/create/index.test.ts [hunk 1]
```diff
@@ -131,120 +115,3 @@ describe('applyHostNameToSitecoreConfig', () => {
...
-    expect(result).to.not.match(
-      /<app ((.|\n|\r)*?)graphQLEndpoint="\/somewhere\/else\/Bar"/,
-      'app graphQLEndpoint'
-    );
-    expect(result).to.not.match(
-      /<BarGraphQLEndpoint ((.|\n|\r)*?)url="\/somewhere\/else\/Bar"/,
-      'GraphQL endpoint'
-    );
-  });
-});
```
Comment (ambrauer) [inline packages/sitecore-jss-cli/src/create/index.test.ts:250]: These replaceName tests are still valuable, no? Even though `applyNameToSitecoreConfig` is gone, would still be good to test against Sitecore config xml file using `applyNameReplacement`.

**Review by ambrauer — APPROVED** [review]
Approved, just see comment about tests which I think would be good to bring back.

File: packages/sitecore-jss-cli/src/create/index.test.ts (hunk 1 above)
Comment (illiakovalenko) [reply packages/sitecore-jss-cli/src/create/index.test.ts:250]: Reverted back, good idea

**Pull request merged**